### PR TITLE
feat: Added glue crawlers

### LINKED ISSUE #1298: AWS::Glue::Crawler

### PR DESCRIPTION
**Pull request by amanenk**
#### Summary

closes #1298 

---

Use the following steps to ensure your PR is ready to be reviewed

- [x] Read the [contribution guidelines](../blob/main/CONTRIBUTING.md) 🧑‍🎓
- [x] Run `go fmt` to format your code 🖊
- [x] Lint your changes via `golangci-lint run` 🚨 (install golangci-lint [here](https://golangci-lint.run/usage/install/#local-installation))
- [x] Update or add tests. Learn more about testing [here](https://docs.cloudquery.io/docs/developers/sdk/testing) 🧪
- [x] Update the docs by running `go run ./docs/docs.go` and committing the changes 📃
- [x] If adding a new resource, add [relevant Terraform files](../terraform) in a separate PR 📂
- [x] Ensure the status checks below are successful ✅

### CODE REVIEW

**Comment by hermanschaaf** [comment]
@amanenk looks like the branch has a couple of merge conflicts, would you mind resolving those? :bow:

File: go.mod [hunk 1]
```diff
@@ -101,6 +102,7 @@ require (
 	github.com/agext/levenshtein v1.2.3 // indirect
 	github.com/agnivade/levenshtein v1.0.1 // indirect
 	github.com/apparentlymart/go-textseg/v13 v13.0.0 // indirect
+	github.com/aws/aws-sdk-go v1.17.7
```
Comment (hermanschaaf) [inline go.mod:105]: Shouldn't we be using `github.com/aws/aws-sdk-go-v2` now? Are glue crawlers not supported there?

**Review by hermanschaaf — APPROVED** [review]
Left one question, but otherwise LGTM!

File: go.mod (hunk 1 above)
Comment (amanenk) [reply go.mod]: `go mod tidy` fixed it

**Review by hermanschaaf — APPROVED** [review]

**Pull request merged**